FUNDING.yml

```
@@ -1,6 +1,6 @@
 # These are supported funding model platforms
 
-github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+github: [withastro]  # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
```

```suggestion
github: withastro
```
This is also valid if you don't plan on having more sponsors